main/ertp/api/amount-math.md

```diff
@@ -1,189 +1,297 @@
+const listAmountA = itemsAmountMath.make(harden['1','2','4']);
+const listAmountB = itemsAmountMath.make(harden['3']);
 
-// Returns ['1', '2', '4', '3']
-const combinedList = amountMath.add(listAmountA, listAmountB);
+// Returns an amount whose value is ['1', '2', '4', '3']
+const combinedList = itemsAmountMath.add(listAmountA, listAmountB);
 ```
 
 ## amountMath.subtract(leftAmount, rightAmount)
 - `leftAmount` `{Amount}`
 - `rightAmount` `{Amount}`
 - Returns: `{Amount}`
 
-Returns a new amount that is the leftAmount minus the rightAmount (i.e. everything in the leftAmount that is not in the rightAmount). If leftAmount doesn't include rightAmount (subtraction results in a negative), throw an error. Because the left amount must include the right amount, this is NOT equivalent to set subtraction.
+Returns a new `amount` that is the `leftAmount` minus the `rightAmount` (i.e. everything in the `leftAmount` that is not in the `rightAmount`). If `leftAmount` doesn't include `rightAmount` (subtraction results in a negative), it throws an error. Because `leftAmount` must include `rightAmount`, this is **not** equivalent to set subtraction.
+
+`leftAmount` and `rightAmount` must be of the same `brand`.
+
+If the `rightAmount` is empty, it returns the `leftAmount`. If both arguments are
+empty, it returns an empty `amount`.
 
 ```js
-const { amountMath } = makeIssuerKit('myItems', 'strSet');
-const listAmountA = amountMath.make(harden['1','2','4']);
-const listAmountB = amountMath.make(harden['3']);
-const listAmountC = amountMath.make(harden['2']);
+import { MathKind, makeIssuerKit } from '@agoric/ertp';
+const { amountMath: itemsAmountMath } = makeIssuerKit('myItems', MathKind.STRING_SET);
+const listAmountA = itemsAmountMath.make(harden['1','2','4']);
+const listAmountB = itemsAmountMath.make(harden['3']);
+const listAmountC = itemsAmountMath.make(harden['2']);
 
 // Returns ['1', '4']
-const subtractedList = amountMath.subtract(listAmountA, listAmountC)
+const subtractedList = itemsAmountMath.subtract(listAmountA, listAmountC)
 
 // Throws error
-const badList = amountMath.subtract(listAmountA, listAmountB)
+const badList = itemsAmountMath.subtract(listAmountA, listAmountB)
 ```
+## Related Methods
+
+The following methods on other ERTP components and objects also either operate
+on or return a brand. While a brief description is given for each, you should
+click through to a method's main documentation entry for full details on
+what it does and how to use it.
+- [`issuer.getAmountOf(payment)`](./issuer.html#issuer-getamountof-payment)
+  - Returns the `amount` description of the `payment`
+- [`issuer.getAmountMathKind()`](./issuer.html#issuer-getamountmathking)
+  - Returns the kind of the `issuer`'s associated `amountMath`.
+- [`zcf.getAmountMath(brand)`](https://agoric.com/documentation/zoe/api/zoe-contract-facet.html#zcf-getamountmath-brand)
+  - Returns the `amountMath` associated with the `brand`.
+
```

Comment:
can we use router-link here too? If we use absolute links like https://agoric.com, there's a couple of downsides. 1) it's fragile if we move the files around. 2) it shows the icon for an "external link" on the page.